src/components/app/content/cardList/WithLoadingDelay.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import "./index.css"
+
+export default function WithLoadingDelay(Card) {
+    return ({isLoading, ...props}) => {
+        if (isLoading) {
+            return (<div className="card card-layout lds-container" key={props.key}>
+                <div className="lds-default">
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                </div>
+            </div>)
+        }
+        return (<Card cardData={props.cardData}
+                      saveCardData={props.saveCardData}
+                      readOnly={props.readOnly}
+                      checkedControl={props.checkedControl}
+                      checked={props.checkedCardIds.includes(props.cardData.id)}
+                      key={props.key}/>);
+    }
```

Comment:
Ideally - to get rid of a direct dependencies to other components. Since Card is a wrapped component - you can return it directly with all the rest props.
`<WrappedComponent ...props> />`
If do everything correctly - your component could be then reused by any other component in order to get 2s loading delay with spinner displayed.